openmetadata-ui/src/main/resources/ui/cypress.config.ts

```diff
@@ -13,6 +13,7 @@ export default defineConfig({
   e2e: {
     // We've imported your old cypress plugins here.
     // You may want to clean this up later by importing these.
+    experimentalSessionAndOrigin: true,
```

Comment:
Please be careful where you are adding. The above comments now pointed to these new lines instead of where they should be please update order of comments

Reply:
Updated